src/Sarif/Autogenerated/Result.cs

```diff
@@ -75,10 +75,10 @@ public SarifNodeKind SarifNodeKind
         public ISet<Stack> Stacks { get; set; }
 
         /// <summary>
-        /// An array of arrays of 'annotatedCodeLocation` objects, each inner array of which comprises a code flow (a possible execution path through the code).
+        /// An array of 'codeFlow' objects relevant to the result.
         /// </summary>
         [DataMember(Name = "codeFlows", IsRequired = false, EmitDefaultValue = false)]
-        public IList<IList<AnnotatedCodeLocation>> CodeFlows { get; set; }
+        public ISet<CodeFlow> CodeFlows { get; set; }
```

Comment:
Here's the object model change. `CodeFlows` is a set of objects of type `CodeFlow`, rather than a list of list of `AnnotatedCodeLocation`. Exactly parallel with `Stacks`.